google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/main/java/com/google/cloud/spanner/jdbc/SingleUseTransaction.java

```diff
@@ -365,23 +363,23 @@ public Long call() throws Exception {
 
     @Override
     public T call() throws Exception {
-      txManager = dbClient.transactionManager();
-      // Check the interrupted state after each (possible) round-trip to the db to allow the
-      // statement to be cancelled.
-      checkInterrupted();
-      try (TransactionContext txContext =
-          txManager.getState()
-                  == com.google.cloud.spanner.TransactionManager.TransactionState.ABORTED
```

Comment:
The `TransactionManager` is always (re-)created at the beginning of this method and closed at the end of the method, so it can never be aborted at this point, so the whole check was redundant anyways.